src/components/06_wrappers/Default/Default.js

```diff
@@ -0,0 +1,130 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { css } from 'emotion';
+import { scaleRotate as Menu } from 'react-burger-menu';
+import { Link } from 'react-router-dom';
+
+let styles;
+
+const menuLinks = {
+  home: [{ path: '/', text: '🏠 Home' }],
+  permissions: [{ path: '/admin/people/permissions', text: '🔐 Permissions' }],
+  content: [
```

Comment:
I'm a bit confused that you are not using the same structure as the PR in https://github.com/jsdrupal/drupal-admin-ui/pull/23

Reply:
I hadn't seen that yet, this is only supposed to be temporary as noted

Reply:
Sure, it'd be nice to come up with a usecase based structure on the other PR though

Reply:
I was going to wait until the other PR is committed and then create a new one to use the new endpoint

Reply:
Fair :) I also just asked @tedbow to maybe think about a better structure than what is right now in the other PR, so its good to not be blocked on this right now.

Reply:
@justafish @dawehner I mentioned that we should just merge the menu pr now so we can continue to quickly iterate on this.

Reply:
It might be worth it to add some sort of normalizer to the returned JSON from #23 to get to this point?

Reply:
:+1: